### ISSUE TITLE
Any docs or wiki on how to?

### ISSUE THREAD

**Issue by sora7672** [MayGo/tockler#284]
Simple as the title said.

Im trying to figure out how to setup the analyser settings or even to understand, what it should do.

From testing i would guess it should create tasks when a specific window is active.
But first i run into the described error with test mode, where the program blanks out
and then i could not figure out how to use task/group/title properly.

I could search in "task" for a titel of a active window. Are "group" and "title" some kind of setters for the output?
When i tryed with search terms it didnt do anything.

Maybe im looking at it wrong, then let me know.
But i would love to have some simple examples on how to set that up.
My goal was to track my learning of different programming languages how long it takes to understand them,
so grouping different programs with specific window titles is my goal here :)

many thanks!

**Comment by MayGo** [comment]
No docs, sorry.
I haven't used this functionality in ages. And I think It is broken, at least it is not working for me.
Idea was, that if you have a analyzer for jira tickets. And you have focused window with jira ticket in title. It will notify you:  Do want to create a new task eg: "Jira - Fixing a bug in main flow". And if you accept, it will create a new Task, where "group" is "app" and "title" is "title". Really confusing naming :) 
<img width="498" alt="Screenshot 2024-05-03 at 22 27 02" src="https://github.com/MayGo/tockler/assets/1929299/312826ed-f256-45fa-a1a6-93c5eb237d2c">

Eg: I want to group all jira tasks with "Jira" and title will be whatever jira ticket title will be.

And if you have opened those windows with specific titles today. You can enable test mode, and it will show regexed results. Input data is not realtime, so if you have opened regexable windows after you have opened settings, then you have to navigate away from settings and back to settings.

<img width="1606" alt="Screenshot 2024-05-03 at 22 22 42" src="https://github.com/MayGo/tockler/assets/1929299/0c1d3bb3-ea81-4f0d-ac96-b371f9f533fc">

**Comment by sora7672** [comment]
Not a problem, i was just thinking, if you really can check for a String inside the title or multiple ones,
if it would create automatically a task based on your analyzer setting it would be a super awesome tool to track what you do :)

(for me e.g. i check sometimes to much in discord or watch animes :D) With that functionality, you could really track and see what you did, without "your brain thinking" you did much, when you didnt.

So in short, if you want to fix that, then dont focus it on Jira only please, rather a setup for in title search :)
